cache-generator/src/test/java/io/ballerina/gencache/cmd/GenCacheCommandTest.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package io.ballerina.gencache.cmd;
+
+import org.testng.annotations.Test;
+import picocli.CommandLine;
+
+@Test
+public class GenCacheCommandTest {
+    public void getExecuteCommand(String path, String workingDir) {
+        String[] args = {"--path", path, "--args", workingDir};
+        GenCacheCmd cmd = new GenCacheCmd();
+        new CommandLine(cmd).parseArgs(args);
+
+        try {
+            cmd.execute();
+        } catch (Exception e) {
+            System.out.println("exception occurred in cache generation " + e);
+        } catch (Error e) {
+            System.out.println("error occurred in cache generation " + e);
+        }
+    }
+}
```

Comment:
```suggestion
}

```

Lets add new line in the end